web/packages/teleport/src/Discover/ConnectMyComputer/SetupConnect/SetupConnect.story.tsx

```diff
@@ -22,34 +22,105 @@ import {
 } from 'shared/components/OverrideUserAgent';
 
 import { ContextProvider } from 'teleport';
+import cfg from 'teleport/config';
 import { UserContext } from 'teleport/User/UserContext';
 import { createTeleportContext } from 'teleport/mocks/contexts';
 import { makeDefaultUserPreferences } from 'teleport/services/userPreferences/userPreferences';
 
 import { SetupConnect } from './SetupConnect';
 
+const { worker, rest } = window.msw;
+
+const oneDay = 1000 * 60 * 60 * 24;
+
+const setupConnectProps = {
+  prevStep: () => {},
+  nextStep: () => {},
+  // Set high default intervals and timeouts so that stories don't poll for no reason.
+  pingInterval: oneDay,
+  showHintTimeout: oneDay,
+};
+
 export default {
   title: 'Teleport/Discover/ConnectMyComputer/SetupConnect',
+  decorators: [
+    Story => {
+      worker.resetHandlers();
+      return <Story />;
+    },
+  ],
+};
+
+const workerNoNodes = () => {
+  worker.use(
+    rest.get(cfg.api.nodesPath, (req, res, ctx) => res(ctx.json({ items: [] })))
+  );
+};
+
+export const macOS = () => {
+  workerNoNodes();
+  return (
+    <OverrideUserAgent userAgent={UserAgent.macOS}>
+      <Provider>
+        <SetupConnect {...setupConnectProps} />
+      </Provider>
+    </OverrideUserAgent>
+  );
+};
+
+export const Linux = () => {
+  workerNoNodes();
+  return (
+    <OverrideUserAgent userAgent={UserAgent.Linux}>
+      <Provider>
+        <SetupConnect {...setupConnectProps} />
+      </Provider>
+    </OverrideUserAgent>
+  );
+};
+
+export const Polling = () => {
+  workerNoNodes();
+
+  return (
+    <Provider>
+      <SetupConnect {...setupConnectProps} />
+    </Provider>
+  );
 };
 
-export const macOS = () => (
-  <OverrideUserAgent userAgent={UserAgent.macOS}>
+export const PollingSuccess = () => {
```

Comment:
Did you open the storybook at localhost:9002? For msw mocks to work, you need to open Storybook at your local IP address and not localhost. I don't remember if that's a quirk of msw or of our implementation.

Reply:
Yeah, I opened localhost.

Reply:
I rewrote the stories to use msw from the package because in the next PR I need to use `ctx.delay` (see #34450), which means they now work over localhost but not over local IP.